docs/subsystems/release-checklist.md

```diff
@@ -51,6 +51,9 @@ preparing a new release.
 * Post the release by [editing the latest tag on
   GitHub](https://github.com/zulip/zulip/tags); use the text from
   `changelog.md` for the release notes.
+
+  **Note:** This will trigger the [GitHub action](https://github.com/zulip/zulip/tree/master/tools/oneclickapps/README.md) for updating DigitalOcen one-click app image. The
+  action uses the latest release tarball published in `zulip.org` for creating the image.
```

Comment:
```suggestion
  **Note:** This will trigger the [GitHub action](https://github.com/zulip/zulip/tree/master/tools/oneclickapps/README.md) for updating DigitalOcean one-click app image. The
  action uses the latest release tarball published on `zulip.org` for creating the image.
```
Looks like the link-checker isn't happy with this, though.  I don't know the right way to fix it.

Reply:
It should have been zulip/zulip/**blob**/master/tools/oneclickapps/README.md instead of zulip/zulip/**tree**/master/tools/oneclickapps/README.md. We check whether the GitHub file and directory links are valid by checking locally instead of making request to the GitHub API.